Gemfile.lock

```diff
@@ -431,8 +431,8 @@ GEM
       railties (>= 5.0.0)
     faker (3.5.1)
       i18n (>= 1.8.11, < 2)
-    faraday (2.10.1)
-      faraday-net_http (>= 2.0, < 3.2)
+    faraday (2.11.0)
+      faraday-net_http (>= 2.0, < 3.4)
```

Comment:
Are these dependencies? I only saw googleauth getting updated in the original PR https://github.com/department-of-veterans-affairs/vets-api/pull/19050

Reply:
that's weird... I'll fix that